# List of lighthouses to be included. They do must be rigid
# with respect to each other and the world frame.
lighthouses:
  - "lighthouse_a"
  - "lighthouse_b"

lighthouse_a:
  serial: "907388239"
  translation : [1.5 0 0] 
  rotation : [-0.5  0.5 0.5 -0.5]

lighthouse_b:
  serial: "3097796425"
  translation : [-1.5 0 0] 
  rotation : [0.5  0.5  0.5  0.5]

# List of trackers to be included. They must be rigid with
# respect to each other and the body being tracked
trackers:
  - "port"
  - "stbd"

port:
  serial: "LHR-08DDBC05"
  translation : [0 0 0] 
  rotation : [0 0 0 1]

stbd:
  serial: "LHR-08DE340B"
  translation : [0 0 0] 
  rotation : [0 0 0 1]

# If the body-frame estimate is available, then use this
# information to periodically correct the estimate pose
frames:
  parent: "world"
  child: "body"

# Optimizer parameters
parameters:
  max_time: 10.0
  max_iterations: 1000
  gradient_tolerance: 1e-6
  threads: 4
  debug: false